src/topology/algebra/polynomial.lean

```diff
@@ -2,47 +2,119 @@
 Copyright (c) 2018 Robert Y. Lewis. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Author: Robert Y. Lewis
-
-Analytic facts about polynomials.
 -/
-import topology.algebra.ring
+import data.polynomial.algebra_map
 import data.polynomial.div
-import data.real.cau_seq
-
-open polynomial is_absolute_value
-
-lemma polynomial.tendsto_infinity {α β : Type*} [comm_ring α] [linear_ordered_field β]
-  (abv : α → β) [is_absolute_value abv] {p : polynomial α} (h : 0 < degree p) :
-  ∀ x : β, ∃ r > 0, ∀ z : α, r < abv z → x < abv (p.eval z) :=
-degree_pos_induction_on p h
-  (λ a ha x, ⟨max (x / abv a) 1, (lt_max_iff.2 (or.inr zero_lt_one)), λ z hz,
-    by simp [max_lt_iff, div_lt_iff' ((abv_pos abv).2 ha), abv_mul abv] at *; tauto⟩)
-  (λ p hp ih x, let ⟨r, hr0, hr⟩ := ih x in
-    ⟨max r 1, lt_max_iff.2 (or.inr zero_lt_one), λ z hz, by rw [eval_mul, eval_X, abv_mul abv];
-        calc x < abv (p.eval z) : hr _ (max_lt_iff.1 hz).1
-        ... ≤ abv (eval z p) * abv z : le_mul_of_one_le_right
-          (abv_nonneg _ _) (max_le_iff.1 (le_of_lt hz)).2⟩)
-  (λ p a hp ih x, let ⟨r, hr0, hr⟩ := ih (x + abv a) in
-    ⟨r, hr0, λ z hz, by rw [eval_add, eval_C, ← sub_neg_eq_add];
-      exact lt_of_lt_of_le (lt_sub_iff_add_lt.2
-        (by rw abv_neg abv; exact (hr z hz)))
-        (le_trans (le_abs_self _) (abs_abv_sub_le_abv_sub _ _ _))⟩)
+import topology.metric_space.cau_seq_filter
+
+/-!
+# Polynomials and limits
+
+In this file we prove that
+
+* `polynomial.eval₂`, `polynomial.eval`, and `polynomial.aeval` define continuous functions;
+* `λ x, ∥polynomial.eval (z x) p∥` tends to infinity provided that `λ x, ∥z x∥` tends to infinity
```

Comment:
Does it make sense to explicitly list the name of the lemma that does this?

Reply:
Done.